src/main/scala/encry/consensus/EncrySupplyController.scala

```diff
@@ -12,9 +12,5 @@ object EncrySupplyController {
       Constants.Chain.InitialEmissionAmount).toLong
   } else Constants.Chain.InitialEmissionAmount
 
-  def supplyBoxAt(height: Height): AssetBox =
-    AssetBox(EncryProposition.heightLocked(height), height * 9, supplyAt(height))
-
-  def totalSupplyBoxes: IndexedSeq[AssetBox] =
-    (Constants.Chain.PreGenesisHeight until Constants.Chain.EmissionEpochLength).map(i => supplyBoxAt(Height @@ i))
+  def initialStateBoxes: IndexedSeq[AssetBox] = IndexedSeq(AssetBox(EncryProposition.open, -9, 0))
```

Comment:
Why does `initialStateBoxes ` reside in `EncrySupplyController`? This is used only once and could be moved directly to `generateGenesisUtxoState()`